api/client/volume/client.go

```diff
@@ -135,14 +49,9 @@ func (v *volumeClient) Create(locator *api.VolumeLocator, source *api.Source,
 	return response.Id, nil
 }
 
-// Status diagnostic information
-func (v *volumeClient) Status() [][2]string {
-	return [][2]string{}
-}
-
 // Inspect specified volumes.
 // Errors ErrEnoEnt may be returned.
-func (v *volumeClient) Inspect(ids []string) ([]*api.Volume, error) {
+func (v *VolumeClient) Inspect(ids []string) ([]*api.Volume, error) {
```

Comment:
Not related to this PR, but it seems that the client takes a list of IDs and passes only one to the server (this last on in this slice). The server also expects only one ID. Should we change the implementation of server to take process multiple IDs.. anyway it is returning a list of volume objects.

Reply:
I think what it does here is that it runs through a for loop in line 60 adding them to the URI. I wonder what the max would be 😨 . Maybe you are correct, we should probably change this to a single volume. I'll create an issue to track this.

Reply:
#401 created